lz4/stream/_stream.c

```diff
@@ -0,0 +1,1404 @@
+/*
+ * Copyright (c) 2019, Samuel Martin
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of Steeve Morin nor the names of its contributors may be
```

Comment:
Why is Steeve Morin mentioned here?

Reply:
oops! copy-paste mistake :confused: 
will clean it